### DISCUSSION TITLE
Mass to selected

### DISCUSSION THREAD

**Discussion by s2b-git** [YetiForceCompany/YetiForceCRM#16701] [Feature improvement]
I did changed naming as per gfx below. Thinking "Mass" it a bit awkward so translated it to "Selected". 
:o)

![image](https://user-images.githubusercontent.com/84110093/205205997-44a3d4dd-edf0-4d81-a99a-5b33330f14a2.png)

**Comment by paula-w** [comment]
Hi
Thank you for the suggestion. We decided not to change the current names for these functions, as they're common in apps.
Please let us know if you have any other suggestions for improvements in the system. Thanks again!
